### PR TITLE
Implement sRGB color conversoin

### PR DESCRIPTION
**Pull request by surma**
Operating on a 2000x1500 pixel image, using Lanczos3 as the algorithm on my MBP:

<table>
	<tr>
		<td colspan=2></td>
		<th colspan=2>Premultiplication</th>
	<tr>
	<tr>
		<td colspan=2></td>
		<th>false</th>
		<th>true</th>
	</tr>
	<tr>
		<th rowspan=2>Color space conversion</th>
		<th>false</th>
		<td>594ms</td>
		<td>823ms</td>
	</tr>
	<tr>
		<th>true</th>
		<td>840ms</td>
		<td>938ms</td>
	</tr>
</table>

I also generate the LUT at build time. Sadly, `const` functions (that can be executed at compile time) aren’t powerful enough yet, so I had to resort to good old code generation (see `build.rs`).

### CODE REVIEW

**Comment by jakearchibald** [comment]
@Kornel so, is `getImageData` in sRGB? We've added the transform here, but just wanting to check we aren't transforming something already transformed.

Demo: https://deploy-preview-510--squoosh.netlify.com/

File: src/codecs/resize/options.tsx [hunk 1]
```diff
@@ -138,6 +141,17 @@ export default class ResizerOptions extends Component<Props, State> {
...
             : null
           }
+          {isWorkerOptions(options) ?
+            <label class={style.optionInputFirst}>
+              <Checkbox
+                name="colorspace"
+                checked={options.colorspace}
+                onChange={this.onChange}
+              />
+              Convert from sRGB to RGB before resizing
```
Comment (jakearchibald) [inline src/codecs/resize/options.tsx]: Any objections to just "Linear RGB"?

Comment (surma) [reply src/codecs/resize/options.tsx]: Nope, that works.

Comment (surma) [reply src/codecs/resize/options.tsx]:
```suggestion
              Linear RGB
```

**Comment by surma** [comment]
CC for the correct Kornel, aka @kornelski ;)

**Comment by kornelski** [comment]
LGTM

**Pull request merged**